frontend/server/src/Controllers/Contest.php

```diff
@@ -121,11 +121,11 @@ public static function apiList(\OmegaUp\Request $r): array {
                 'past',
             ]
         );
-        $page = (isset($r['page']) ? intval($r['page']) : 1);
-        $pageSize = (isset($r['page_size']) ? intval($r['page_size']) : 20);
-        $activeContests = isset($r['active'])
-            ? \OmegaUp\DAO\Enum\ActiveStatus::getIntValue(intval($r['active']))
-            : \OmegaUp\DAO\Enum\ActiveStatus::ALL;
+        $page = intval($r['page'] ?? 1);
```

Comment:
```suggestion
        $page = $r->ensureOptionalInt('page') ?? 1;
```

si ya vamos a hacer cambios, mejor hay que migrar al patrón nuevo. y con eso L105-L106 se pueden ir.

Reply:
Oh, TIL